routes/index.js

```diff
@@ -19,4 +22,40 @@ router.get('/registration', function(req, res) {
   res.render('registration.njk');
 });
 
+router.post('/badgein',jsonParser,function(req,res){
+  if(!req.body)
+    { 
+     return res.sendStatus(400);
+    }
+
+  var dbObject  = dbAPI.validateUser(undefined,BadgeNumber);
+  var BadgeNumber = req.body.badgeNumber;
+
+  if(dbObject==undefined)
+   {
+     var url = '/registration/' + BadgeNumber;
+     res.redirect(url);
+   }
+  if(dbObject == true)
```

Comment:
validateUser returns either a userData object or undefined

Reply:
how to get the userData object from the dbm.js file ?